### PR TITLE
PopoverEducational: changing `accessibilityLabel` prop to optional

### PR DESCRIPTION
**Pull request by eniomoura**
### Summary

accessibilityLabel is now not a required prop in PopoverEducational

#### Why?

It shouldn't have been required, given in many cases there is no context to define an appropriate specific accessibilityLabel, and a default is provided by the component.

### CODE REVIEW

File: packages/gestalt/src/PopoverEducational.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ type PrimaryActionType = {|
 |};
 
 function PrimaryAction({
-  accessibilityLabel,
+  accessibilityLabel = 'Popover',
```
Comment (AlbertCarreras) [inline packages/gestalt/src/PopoverEducational.js:32]: hehe not here!

Comment (AlbertCarreras) [reply packages/gestalt/src/PopoverEducational.js:32]: This is the button

Comment (AlbertCarreras) [reply packages/gestalt/src/PopoverEducational.js:32]: In line 123 we should see the default. Can you make sure you rebase?

Screenshot by Dropbox Capture

Comment (eniomoura) [reply packages/gestalt/src/PopoverEducational.js:32]: Argh, got distracted and ended up changing the wrong thing.
Will rebase and fix.

Comment (eniomoura) [reply packages/gestalt/src/PopoverEducational.js]: Done :)

**Review by AlbertCarreras — APPROVED** [review]

**Pull request merged**